cypress/test/campaignTests/AnnouncementHero.cy.js

```diff
@@ -5,34 +5,11 @@ beforeEach(() => {
   mount(<AnnouncementHero />);
 });
 
-
 describe('AnnouncementHero Component', () => {
   it('should render the component when the date is within the valid range', () => {
-    // Set the current date to May
-    const mockDate = new Date('2023-05-01T00:00:00Z');
-    cy.clock(mockDate.getTime());
-    // Assert that the component is rendered 
-    //check for background color 
-    cy.get('[data-testid="AnnouncementHero-main-div"]').should('have.class', 'bg-gray-50');
-  });
-
-  it('should display the correct event information', () => {
-    const mockDate = new Date('2023-05-01T00:00:00Z');
-    cy.clock(mockDate.getTime());
-    // Assert the event details
-    cy.get('[data-testid="AnnouncementHero-main-div"]').contains('AsyncAPI Conf on Tour 2023').should('exist');
-    cy.get('[data-testid="AnnouncementHero-main-div"]').contains('London Edition').should('exist');
-    cy.get('[data-testid="AnnouncementHero-main-div"]').contains('20th of September, 2023 | London, UK').should('exist');
-    cy.contains('Submit a session').should('exist');
-  });
-
-  it('should have a link to submit a session', () => {
-    const mockDate = new Date('2023-05-01T00:00:00Z');
-    cy.clock(mockDate.getTime());
-    // Assert the link
-    cy.get('[data-testid="AnnouncementHero-submit-session"]').should('have.attr', 'href', 'https://conference.asyncapi.com/')
-      .should('have.attr', 'target', '_blank')
-      .contains('Submit a session');
```

Comment:
Why did we remove these tests?

Reply:
For these tests , since now the announcement hero is not shown on the website these would fail , adding or using cy.clock doesn't work here since this component has no specific details of when it is rendered .